lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -829,15 +829,16 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             course_grade.letter_grade,
             student.id in whitelisted_user_ids
         )
-        if certificate_info[0] == 'Y' and course_grade.letter_grade is None and student.id not in whitelisted_user_ids:
+        if certificate_info[0] == 'Y':
             TASK_LOG.info(
                 u'Student is marked eligible_for_certificate despite failing the course '
```

Comment:
you'll want to change this log message, because these students are almost always NOT failing the course